README.md

```diff
@@ -1,8 +1,8 @@
 # k8s-initiator-app
 
-An application that runs as daemonset only in the master nodes to maintain the state of some Kubernetes resources through cluster upgrades or master replacements.
+An application that runs as daemonset only in the master nodes (by default) to maintain the state of some Kubernetes resources through cluster upgrades or master replacements.
```

Comment:
```suggestion
An application that runs as daemonset (masters and/or workers) to maintain the state of some Kubernetes resources through cluster upgrades or master replacements.
```

Reply:
IMO it should run in all nodes by default, and you can customize only workers or master via values.yaml, WDYT?